packages/cx-layout/src/components/WhereToBuy/WhereToBuyClean.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright © 2022 Johnson & Johnson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC } from 'react';
+import { asCxTokenSpec } from '@bodiless/cx-elements';
+import {
+  A,
+  Div,
+  Span,
+  designable,
+} from '@bodiless/fclasses';
+import { CartIcon } from './assets/CartIcon';
+import { WhereToBuyComponents, WhereToBuyProps } from './types';
+
+const socialLinksComponents: WhereToBuyComponents = {
+  Wrapper: Div,
+  Button: A,
+  Icon: CartIcon,
```

Comment:
I wonder if instead of starting with this we should add it in the content section of a default token